jest.config.js

```diff
@@ -23,16 +22,14 @@ const projects = [
     ...commonConfig,
     ...{
       roots: ['./packages/graphql-hooks-memcache'],
-      displayName: 'graphql-hooks-memcache',
-      testMatch: ['<rootDir>/packages/graphql-hooks-memcache/**/*.test.[jt]s']
+      displayName: 'graphql-hooks-memcache'
     }
   },
   {
     ...commonConfig,
     ...{
       roots: ['./packages/graphql-hooks-ssr'],
-      displayName: 'graphql-hooks-ssr',
-      testMatch: ['<rootDir>/packages/graphql-hooks-ssr/**/*.test.[jt]s']
```

Comment:
why are these not needed anymore?

Reply:
the default `testMatch` covers `spec.*`/`test.*` and `js`/`ts`/`jsx`/`tsx` extensions as well as the `__tests__` folder - as `roots` is set, only the appropriate folder will be searched

Reply:
and it only navigate in `roots` then? that was my main question really, to avoid each of these to run ALL the tests in the repo

Reply:
yes that's correct 👍 18 test suite passes/178 test passes in both `master` and this PR